src/provisioner/log.py

```diff
@@ -16,32 +16,68 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
 import os
+
 from cortx.utils.log import Log
+from cortx.provisioner import const
 
 
 class CortxProvisionerLog(Log):
     """ Redirect log message to log file and console using cortx utils logger """
 
+    default_log_path = '/var/log/cortx'
+    app_name = 'provisioner'
     logger = None
 
     @staticmethod
-    def init(service_name, log_path=None, level='INFO', console_output=False):
+    def initialize(service_name, log_path=None, level='INFO', console_output=False):
         """
         Initialize and use cortx-utils logger to log message in file and console.
         If console_output is True, log message will be displayed in console.
         """
+        if not CortxProvisionerLog.logger:
+            log_path = log_path if log_path else CortxProvisionerLog.default_log_path
+            log_path = os.path.join(log_path, CortxProvisionerLog.app_name)
+
+            if level not in ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']:
```

Comment:
Ideally we should define supported log-levels list in constant instead of defining it at multiple places...

Reply:
Moved to const.py

Reply:
Done.